src/main/scala/io/iohk/ethereum/jsonrpc/security/SSLContextFactory.scala

```diff
@@ -0,0 +1,103 @@
+package io.iohk.ethereum.jsonrpc.security
```

Comment:
why not `io.iohk.ethereum.security`? I mean there is no other word  referencing to jsonrpc in this file.